lib/translator/locale-parser.js

```diff
@@ -49,13 +49,13 @@ function getLocales(acceptLanguage) {
  * @param {string} locale
  * @returns {string}
  */
-function normalizeLocale(locale, defaultLocale) {
+function normalizeLocale(locale, defaultLocale, logger = null) {
     try {
         new MessageFormat(locale);
 
         return locale;
     } catch (err) {
-        Logger.log(err);
+        logger ? logger.error(err) : defaultLogger(err);
```

Comment:
This is not a useful error message, so we should just remove it rather than add the complexity of passing this logger around.